tests/test-core/test-block.js

```diff
@@ -0,0 +1,73 @@
+/* globals describe, it */
+
+'use strict'
+
+const expect = require('chai').expect
+const base58 = require('bs58')
+const fs = require('fs')
+const IPFS = require('../../src/ipfs-core')
+const Block = require('ipfs-merkle-dag').Block
+
+const isNode = !global.window
+
+const fileA = isNode
+  ? fs.readFileSync(process.cwd() + '/tests/repo-example/blocks/12207028/122070286b9afa6620a66f715c7020d68af3d10e1a497971629c07606bfdb812303d.data')
+  : new Buffer(require('raw!./../repo-example/blocks/12207028/122070286b9afa6620a66f715c7020d68af3d10e1a497971629c07606bfdb812303d.data'))
```

Comment:
`bug` found (this took me a bit to figure out :))

When using raw-loader, a string is returned with the contents of the file, if the file content was text, it would be ok, but since it is an encoded protobuf, it will add extra encoding data (utf8) adding extra bytes to the content of the Buffer, increasing the size of the Block and by so, failing the test. Need to get another way to load the blocks into localstorage